Casks/molotov.rb

```diff
@@ -1,15 +1,32 @@
 cask "molotov" do
   version "4.5.0"
-  sha256 "9366230de159a4b5dffe29e1823b6e3bc0d6e6fd2ddc2c4f1b9e9764319cf995"
 
-  url "https://desktop-auto-upgrade.molotov.tv/mac/Molotov-#{version}-mac.zip"
+  arch = Hardware::CPU.intel? ? "" : "arm64"
```

Comment:
This would save a lot of duplication
```suggestion
  arch = Hardware::CPU.intel? ? "" : "-arm64"
  arch_folder = Hardware::CPU.intel? ? "" : "m1/"
```

Reply:
Thanks for the suggestion @SMillerDev. I've just pushed a commit that gets rid of the following conditionals.